src/components/Authentication/LoginPage.js

```diff
@@ -125,6 +166,57 @@ const LoginPage = () => {
         }
     };
 
+    function refreshTokenSetup(res) {
+        return new Promise((resolve) => {
+            const refreshToken = async () => {
```

Comment:
Is this to refresh the googleAuth token? if so rename the function to describe it